sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -25,7 +29,10 @@ public CouchConfig getCouchConfig(String db) {
                         " is required to run tests with Cloudant.");
             }
             return config;
-        } else {
+        } else if (SPECIFIED_COUCH) {
+            return SpecifiedCouch.defaultConfig(db);
```

Comment:
I don't really see the value of this SpecifiedCouch class. Why not just let the user pass in a url string and have done with it?